tests/integration/testutils/test_utils.go

```diff
@@ -0,0 +1,320 @@
+//go:build integration
```

Comment:
would make sense to split the content of the file into test_utils and dockercompose_utils?

Reply:
I'll evaluate this when I'm adding the long running-mode integration tests